### DISCUSSION TITLE
I couldn't find any non-5-star reviews

### DISCUSSION THREAD

**Discussion by bu6030** [microsoft/MicrosoftEdge-Extensions#151] [Q&A] [answered]
My Edge Extensions URL is 
https://microsoftedge.microsoft.com/addons/detail/nlekkaghplahbknlhefkempopljodhee

My extension has a rating of 4.3, but after checking all the ratings, they are all 5-stars. I couldn't find any non-5-star reviews. Could you please look into what might be going on?

[Enter feedback here]
![image](https://github.com/MicrosoftDocs/edge-developer/assets/11482988/740b4b2f-62e9-4428-a954-a7cf46976759)
![image](https://github.com/MicrosoftDocs/edge-developer/assets/11482988/2669c943-2610-4126-86e9-2aaeb56377c0)

**Comment by bu6030** [comment]
I want to know the reason, why the average score is lowwer than the excpected score.

**Comment by ManikanthMSFT** [comment]
Hi @bu6030, thank you for reporting this. We are looking into this with our engineering team and will let you know as soon as we have an update.

**Reply by bu6030** [reply]
Thank you very much, waiting for your reply.

**Comment by ManikanthMSFT** [comment]
@bu6030, we appreciate your patience. 

We see that you are currently viewing region-based reviews. If you wish to view all reviews, you can do so from your Partner Center account. This will provide you with a comprehensive view of all feedback received.

We hope this helps, and please don’t hesitate to reach out if you have any further questions or concerns.

**Reply by bu6030** [reply]
![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/11482988/49f23731-1c65-41a0-b3fd-a062834fa4e9)

This is the page of my Partner Center account.   
And the first comments which created by me is also the page of Partner Center account.   
How to change the reaion?

**Comment by ManikanthMSFT** [comment, marked as answer]
Hi @bu6030, we just got an update from our team which they have figured out the current list which developers see only has reviews and not ratings. So, you have received 5 X revies and 1 X rating.

Let me clarify this for you.

Ratings on our platform can be provided with or without accompanying review text. When you see the list of reviews, it only includes those ratings that have been provided with review text. This is why you might see fewer reviews than the total number of ratings.

The average rating that you see is calculated from all ratings, both those with and without review text. This ensures that all customer feedback is considered, providing a more accurate overall rating.

In your case, it’s possible that the 6th rating was provided without any review text, which is why it doesn’t appear in the review list.

I hope this clarifies your query. If you have any more questions, feel free to ask.

**Reply by bu6030** [reply]
I got id.
Thank you very much.
